package.json

```
@@ -13,11 +14,10 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/aichner/React-MDB-Template.git"
+    "url": "https://github.com/snek-at/front.git"
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
```

Add required scripts.

```suggestion
    "build": "react-scripts build",
    "predeploy": "yarn run build",
    "deploy": "gh-pages -d build",
    "eject": "react-scripts eject"
```